packages/demo-cards/pet.gts

```diff
@@ -25,7 +25,11 @@ export class Pet extends Card {
   });
   static embedded = class Embedded extends Component<typeof this> {
     <template>
-      <CardContainer class='demo-card' @displayBoundaries={{true}}>
+      <CardContainer
+        class='demo-card'
+        @displayBoundaries={{true}}
+        ...attributes
```

Comment:
I needed to add `...attributes` to forward the element modifier, but this is only for `Pet`. We need to add this to other embedded cards as well. I'm torn between adding `...attributes` to other embedded demo cards, or finding a solution where this is added to embedded cards automatically. What do you think?

Reply:
it seems like the framework should probably do this. the less boilerplate the card author needs to perform the better, IMHO.